transcode.py

```diff
@@ -1,64 +1,80 @@
 import logging
 import os
-from typing import Optional
+import time
 
-import base_util
-from config import data_base_dir
-from config import ae_file_extension, ae_convert_to_mono, ae_samplerate_hz
+from base_util import (
+    Provenance,
+    run_shell_command,
+    remove_all_input_output,
+    run_shell_command_with_output,
+)
+from config import ae_file_extension
 
 logger = logging.getLogger(__name__)
 
 
-def ffmpeg_transcode(input_path, asset_id, extension) -> Optional[str]:
+def ffmpeg_audio_extraction(
+    input_file: str,
+    asset_id: str,
```

Comment:
I feel asset_id and extension should be obtained within this function, rather than passed as an argument